src/extended_json.ts

```diff
@@ -190,6 +190,17 @@ function getISOString(date: Date) {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function serializeValue(value: any, options: EJSONSerializeOptions): any {
+  if (value instanceof Map || isMap(value)) {
+    const obj: Record<string, unknown> = {};
```

Comment:
```suggestion
    const obj: Record<string, unknown> = Object.create(null);
```

(otherwise this doesn’t handle `new Map([['__proto__', { a : 1 }]])` & friends well)

Reply:
Thanks!